packages/pwa/app/utils/utils.js

```diff
@@ -158,13 +158,90 @@ export const capitalize = (text) => {
 }
 
 /**
- * Get the pwa configuration object from pwa-kit.config.json
- * @returns {object} - the configuration object
+ * This function return the identifiers (site and locale) from the given url
+ * The site will always go before locale if both of them are presented in the pathname
+ * @param path {string}
+ * @param urlConfig {object}
+ * @returns {object}
  */
-export const getConfig = () => pwaKitConfig
+export const getParamsFromPath = (path, urlConfig = {}) => {
```

Comment:
Thank you for the feedback, We are currently looking into it to provide more flexibility on that.

Reply:
Hello,
i had not tested it yet, but is this case possible?

- domain.de/de - site=MySiteEU, locale=en_DE
- domain.de/en - site=MySiteEU, locale=en_DE
- domain.at/de - site=MySiteEU, locale=de_AT
- domain.at/en - site=MySiteEU, locale=en_AT
- domain.gb/en - site=MySiteGB, locale=en_GB

In this example i have for example one site for European Union, which has multiple Locale and i want for every different country his own domain. And for GB then for example a new Site because of specific differences to EU and also the country with his own domain.

Did i have the flexibility of a combination of domain and site alias to handle this scenario?

Reply:
just a quick question? domain.de/de => is the `de` site or locale? if it is locale, then the locale is set to path, site is `none`. if not, it is the other way around